src/20250623_AaveV3Ethereum_AdditionOfUSDeToEthenaPrincipalTokenStablecoinEModes/AdditionOfUSDeToEthenaPrincipalTokenStablecoinEModes.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Addition of USDe to Ethena Principal Token Stablecoin E-Modes"
+author: "Aave-Chan Initiative"
+discussions: "Addition of USDe to Ethena Principal Token Stablecoin E-Modes"
```

Comment:
Change it to: https://governance.aave.com/t/arfc-addition-of-usde-to-ethena-principal-token-stablecoin-e-modes/22355